remotetracker/tracker.go

```diff
@@ -0,0 +1,15 @@
+package remotetracker
+
+// Tracker represents a remote issue tracker
+// for e.g github, trello, jira, bugzilla
+type Tracker interface {
+	Fetch(query string) error
+	Import() error
```

Comment:
Database storage should not be a concern of the 'RemoteWorkItem Provider'. The 'Provider' should only care about Getting the data from the Remote system.